pkg/proxy/nftables/proxier.go

```diff
@@ -0,0 +1,1542 @@
+/*
+Copyright 2015 The Kubernetes Authors.
```

Comment:
Yeah, I don't know either.

In this case the copyright got copied over because the commit "`Add a dummy nftables kube-proxy backend which is just a copy of iptables`" is literally just

```
mkdir pkg/proxy/nftables
cp pkg/proxy/iptables/proxier*.go pkg/proxy/nftables/
perl -pi -e 's/^package iptables/package nftables/;' pkg/proxy/nftables/*.go
```

There are probably still some references to iptables in nftables/proxier.go at the end of the branch that I have to clean up...